Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -1231,6 +1234,35 @@ open class ComposerVC: _ViewController,
         content.attachments.append(attachment)
     }
 
+    /// The maximum upload file size depending on the attachment type.
+    ///
+    /// The max attachment size can be set from the Stream's Dashboard App Settings.
+    /// If it is not set, it fallbacks to the deprecated `ChatClientConfig.maxAttachmentSize`.
+    /// - Parameter attachmentType: The attachment type that is being uploaded.
+    /// - Returns: The file size limit in bytes. The default value is 100MB.
+    open func maxAttachmentSize(for attachmentType: AttachmentType) -> Int64 {
```

Comment:
That is not a problem because if a customer sets a value in the Dashboard, it means he does not care anymore about `maxAttachmentSize`. Only if nothing is changed in the Dashboard, is where we use the deprecated flag.

Reply:
depends on whether the dashboard values are nil or have a default value. If the former is the case, then we are ok. Otherwise, we need to update the logic.

Reply:
Yeah the default value is 0, so we are good here 👍